src/quick-lint-js/parse.h

```diff
@@ -1549,6 +1576,11 @@ class parser {
         method_attributes = function_attributes::async_generator;
         this->skip();
       }
+      if (this->peek().type == token_type::kw_static) {
+        // async static method() {}  // Invalid
+        // async static() {}
```

Comment:
Is `class { async static() {} }` already tested by an existing test? Or should we add a test for it?

Reply:
I believe this test `class_statement_with_keyword_property` handles this.